README.md

```diff
@@ -2,58 +2,88 @@
 
 # Project Name
 
-> Description the project.
-
+> Personal Portfolio project
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+- HTML/CSS
+- NPM, Git, Github
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
+**This project includes linters for Reporting errors in HTML, CSS and An open-source, automated tool for improving the quality of web pages, which has audits for performance, accessibility, progressive web apps, SEO and more. It is a summary of all my works and known technologies**
 
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
+1. Node package Manager(NPM)
+   - To install NPM on your local machine, please visit [ NPM official website](https://nodejs.org/en/download/)
+
+- Text editor(Vscode, Atom, NotePad++, Sublime)
+
 ### Setup
 
+Set up folder as:
+
+/Portfolio-project
+-.github/workflows - linters.yml
+
+- Package.json
+- index.html
+- /CSS
+  - style.css
+- /img
+  -/icons
+  - angellist.svg
+  - circle 2 header.svg
+  - down-arrow.svg
+  - email.png
```

Comment:
Please remove this file. This file was not used anywhere on the mobile version design you already made. So to make the project even smaller, remove unused files.